db/migrate/20190617191109_move_awx_credentials_to_authentications.rb

```diff
@@ -267,6 +271,15 @@ def up
     end
   rescue PG::ConnectionBad
     say("awx database is unreachable, credentials cannot be migrated")
+  rescue Fernet256::InvalidToken
+    say("awx password invalid, credentials cannot be migrated")
```

Comment:
AWX should be capitalized through these sentences.

Reply:
👍 Fixed.